src/main/java/conan/actions/ActionUtils.java

```diff
@@ -30,7 +37,11 @@
      * @param update    true if it's update and install action.
      */
     static void runInstall(Project project, Component component, boolean update) {
-        new Version(project).run();
+
+        if(!isConanInstalled(project)){
```

Comment:
White-space should be added after the "if" to be compatible with the existing code conventions.
Let's also remove the first new-line in the method.